modules/cache/src/main/java/org/eclipse/xpanse/modules/cache/RegionAzCacheConfig.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.modules.cache;
+
+import com.github.benmanes.caffeine.cache.Caffeine;
+import java.util.concurrent.TimeUnit;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.cache.caffeine.CaffeineCacheManager;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Region AZ information cache configuration class.
+ */
+@Configuration
+public class RegionAzCacheConfig {
+
+    @Value("${caffeine.cache.region.az.expire.time:60}")
+    private long duration;
+
+    /**
+     * Create the configured CaffeineCacheManager.
+     *
+     * @return CaffeineCacheManager
+     */
+    @Bean
+    public CaffeineCacheManager caffeineCacheManager() {
+        CaffeineCacheManager cacheManager = new CaffeineCacheManager("region_azs_cache");
```

Comment:
can we move this cache manager name to a constants file in cache module? then it can be referred to from other modules instead of rewriting this string in multiple places.